core/src/test/java/org/javarosa/core/form/api/test/TextFormTests.java

```diff
@@ -17,13 +17,11 @@
 import org.javarosa.form.api.FormEntryController;
 import org.javarosa.form.api.FormEntryPrompt;
 
-import j2meunit.framework.Test;
-import j2meunit.framework.TestCase;
-import j2meunit.framework.TestMethod;
-import j2meunit.framework.TestSuite;
-
-public class TextFormTests extends TestCase {
+import org.junit.Test;
+import org.junit.Before;
+import static org.junit.Assert.*;
 
+public class TextFormTests {
```

Comment:
The tests in this file were never being called! Enabling them, it looks like 3 of them are failing